addons/calendar/models/calendar_event.py

```diff
@@ -431,6 +431,12 @@ def _check_closing_date(self):
                     )
                 )
 
+    def _check_organizer_validation_conditions(self, vals_list):
+        """ Method for check in the microsoft_calendar module that needs to be
+            overridden in appointment.
+        """
```

Comment:
See PR linked below, sadly there's no bridge for appointment and microsoft_calendar so we had to add the method in base calendar

As it's effectively only used in microsoft_calendar I think it may be worth mentionning

Also thanks for having a look!